### PR TITLE
refactor: remove verification request link and new badge in interests

### PR DESCRIPTION
**Pull request by DanIsNearby**
### <samp>🤖 Generated by Copilot at 008b191</samp>

This pull request updates the translations and UI components of the web app to align with the new profile NFT feature. It removes and changes some translations in various languages, and uses the term `burn` instead of `delete` for the NFT operation. It also disables the verification feature and removes a placeholder component for adding new interests.

## Related issues

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### <samp>🤖 Generated by Copilot at 008b191</samp>

*  Disable verification feature by removing link to request verification from verification settings component ([link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-eea0b40850062bd6b620dc5bb1fbab6860e6ec837c6129e4335f9c7c0725da90L25-L31)) and removing translations for "Request Verification" message from Spanish, Russian, Tamil, Ukrainian, and Chinese locales ([link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-76ad6152acaaf4ecb26eb04e2fbe8c4600e13a691fb974da91db3ebd52ca9294L1760-L1763), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-e53d289af6055d499884e2a85fd1686e0115f12d3a623db33ec92daf6c1ee7feL1760-L1763), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-74c63dfeab2412f39f5a30f665845db007129732f3c107216bfb25faa66e3a48L1760-L1763), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-cf775f0448152a93d62b1457d3ad054d2b089160e83572be74dbafb5c024c26dL1760-L1763), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-be3912a3e9938530b5702e29149c1ee5dc96cd68c54b3691dd9d5eb225630ccdL1760-L1763))
*  Use consistent terminology for profile NFT operation by replacing "delete" with "burn" in delete settings component ([link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-24350b827973549fb283ddcf1c5a7ee4401f88d7a72d2205a78a1a0350c80cd6L123-R123)) and updating translations for "burn" message in English, Spanish, French, Russian, Tamil, Ukrainian, and Chinese locales ([link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-a91393d1e62c7dd6f423054d58557fdef61bb5a84cc28248690ad03e1b160365L1887-R1888), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-76ad6152acaaf4ecb26eb04e2fbe8c4600e13a691fb974da91db3ebd52ca9294L1887-R1883), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-e6e55bf08ac014b03ae04b5978c2232855f354712cd50cfe8942a7d6c84635d6L1887-R1887), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-e53d289af6055d499884e2a85fd1686e0115f12d3a623db33ec92daf6c1ee7feL1887-R1883), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-74c63dfeab2412f39f5a30f665845db007129732f3c107216bfb25faa66e3a48L1887-R1883), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-cf775f0448152a93d62b1457d3ad054d2b089160e83572be74dbafb5c024c26dL1887-R1883), [link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-be3912a3e9938530b5702e29149c1ee5dc96cd68c54b3691dd9d5eb225630ccdL1887-R1883))
*  Simplify UI for interests settings by removing non-functional `New` component ([link](https://github.com/lensterxyz/lenster/pull/3059/files?diff=unified&w=0#diff-c93ba58b3e484f4c8768db320e2fb2118db6acfb3915ebe0c00ff6f4c56c88f0L42))

## Emoji

🔥🚫🗑️

### CODE REVIEW

**Comment by DanIsNearby** [comment]
This pull request changes some of the stuff in Lenster to make it more "Logicall" i also plan to make a suggestion to switch the naming of commenting acess as right all of the setting buttons are saying what they do and the comment is only one stating what are the setting set so i am planing to suggest a change of the "Acess" button to "Visibility" and the "Comment Settings" button into Acess

**Comment by DanIsNearby** [comment]
Also the post actions are non consistant everything has action count but likes have "Like" "Unl;ike" so it should go either the way of chaning like into likes count or changing everything other to an action name

**Review by bigint — COMMENTED** [review]
Make sure to remove imports and follow lint rules, I fixed them manually on the PR, make sure to remove it from next ones 😊

File: apps/web/src/components/Settings/Delete/index.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ const DeleteSettings: FC = () => {
           </div>
           <p>
             <Trans>
-              This will permanently delete your Profile NFT on the Lens
+              This will permanently burn your Profile NFT on the Lens
```
Comment (bigint) [inline apps/web/src/components/Settings/Delete/index.tsx:123]: Burn doesn't makes sense for normal users, so I'll leave it to 'delete'

**Review by bigint — CHANGES_REQUESTED** [review]
@DanIsNearby requested some changes

**Comment by bigint** [comment]
Also I request to follow conventional PR names for better understanding 🙇🏼

File: apps/web/src/components/Settings/Delete/index.tsx (hunk 1 above)
Comment (DanIsNearby) [reply apps/web/src/components/Settings/Delete/index.tsx]: Kk i have changed it back

**Comment by DanIsNearby** [comment]
@bigint I have changed "burn" back to "delete" any other changes i should make?

**Review by bigint — APPROVED** [review]
**LGTM** :shipit: 

Rad LGTM

**Pull request merged**